components/identifier-generator/front/src/feature/tabs/__tests__/FamilyLine.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {fireEvent, render, screen} from '../../tests/test-utils';
+import {FamilyLine} from '../conditions';
+import {CONDITION_NAMES} from '../../models';
```

Comment:
```suggestion
import {CONDITION_NAMES, Operator} from '../../models';
```

why don't use Operator.IN ?